config/locales/custom/activerecord.yml

```diff
@@ -26,4 +28,5 @@ en:
 
     attributes:
       budget/investment:
-        heading_id: "Category"
\ No newline at end of file
+        estimated_price: Investment author’s estimated price
```

Comment:
We usually use `'` instead of `’`. Besides, wouldn't "estimated price" be enough? In this context, we're showing a form to the author, so using the term "Investment author" sounds redundant :thinking:.

Should we show the currency in the label, like "Estimated price (€)" (taken from the budget's currency)?